configs/maventa.json

```diff
@@ -1,22 +1,32 @@
 {
   "index_name": "maventa",
   "start_urls": [
-    "https://documentation.maventa.com/"
+    "https://documentation.maventa.com/",
```

Comment:
Should be the last as it will always match first all URLs

Reply:
moved to last